services-collections.md

```diff
@@ -192,14 +337,23 @@ You may also pass a key / value pair to the `contains` method, which will determ
 
 Finally, you may also pass a callback to the `contains` method to perform your own truth test:
 
-    $collection = new Collection([1, 2, 3, 4, 5]);
+    $collection = collect([1, 2, 3, 4, 5]);
 
-    $collection->contains(function ($key, $value) {
+    $collection->contains(function ($value, $key) {
```

Comment:
When did these parameters switch?

Reply:
I am guessing it is an error in the official docs,quick glance at the code shows it should be key, value

https://github.com/laravel/framework/blob/6.x/src/Illuminate/Support/Collection.php#L171

Reply:
Internally it passes the closure to Arr::first() which is $value, $key: https://laravel.com/docs/6.x/helpers#method-array-first

Reply:
Looks like it happened in 5.3, so if there are any breaking changes from this they would have already happened in the 5.5 update so this is fine: https://github.com/laravel/framework/commit/99ca947f7362a105d61f401d3e45dc928bab783c